primitives/application-crypto/src/traits.rs

```diff
@@ -31,7 +31,7 @@ use sp_std::{fmt::Debug, vec::Vec};
 /// Typically, the implementers of this trait are its associated types themselves.
 /// This provides a convenient way to access generic information about the scheme
 /// given any of the associated types.
-pub trait AppCrypto: 'static + Send + Sync + Sized + CryptoType + Clone {
+pub trait AppCrypto: 'static + Send + Sized + CryptoType + Clone {
```

Comment:
Unfortunately the bandersnatch_vrfs `SecretKey` backend as is is not `Sync` so I had to remove this bound from a couple of places. Doesn't look like we really need this bound BTW

Reply:
We'll hopefully get someone to do constant time arithmetic one day, but right now I don't want to take anyone off anything more important.